### DISCUSSION TITLE
Missing tabs for certain Mapping Tables

### DISCUSSION THREAD

**Discussion by sweco-sehhan** [INSPIRE-MIF/helpdesk#182] [Q&A]
Hi,
I'm working with mapping of data for various Inspire themes. For at least Environmental Managament Facilities and Environmental Monitoring Facilities the mapping tables (excel documents) are missing the Association tab, even though they internally refer to this specific non-existent tab. The other tabs, SelectedObjects, ComplexTypes and CodeListEnumeration, are present in both of these mapping tables.

**Comment by fabiovinci** [comment]
Dear @Hans-Sweco,

thanks for the notification, it is a typo generated by the tool used to generate mapping tables.
In particular, the "Association" tab is present only when there are associations with feature types related to other data models (e.g. Addresses, in which there is an association with Cadastral Parcel).

If all associations refer to feature types defined in the same data model, the "Association" tab is missing since the feature types are described in the "SelectedObjects" tab, as in the case of the two mentioned mapping tables.

Example for Environmental Monitoring Facilities Mapping Table:
![image](https://github.com/INSPIRE-MIF/helpdesk/assets/22212406/4a94a3ec-88ca-4587-a7a2-d2aa4a37bc85)

I hope this helps and answers your question.

**Reply by sweco-sehhan** [reply]
Thanks for the quick response, but...
I actually do miss things in these themes, particularly Environmental Monitoring Facilites that I have studied in detail. As an example, this theme includes the LegislationCitation value/enumeration on the legal Background attribute. I feel this should be in the Associations tab, as is the case for other themes where LegislationCitation is included, e.g. the AM theme. Instead this enumeration is in EF located in the ComplexTypes tab. Why is that? Comparing EF and AM, another difference is that in EF Thematic Identifier is missing from the Complex Types tab, even though it is referred to as an enumeration on the Selected Objects tab.


I have worked thoroughly with the AM theme, and I felt that theme where more coherently documented. Isn't the point that the structure should be the same throughout all themes? Or have I missunderstood something in this more complex theme (EF)?
